src/tests-synchronized-iterator.cc

```diff
@@ -2,10 +2,10 @@
 // Created by Lukáš Holík on 29.10.2022.
 //
 
-#include "../3rdparty/catch.hpp"
+#include "catch.hpp"
 
-#include <mata/util.hh>
-#include <mata/synchronized-iterator.hh>
+#include "mata/util.hh"
+#include "mata/synchronized-iterator.hh"
```

Comment:
Why have you changed this? We use `<>` version in the library and it also looks better.

Reply:
did I do it?

Reply:
yes

Reply:
fixed